fala/templates/adviser/other_region.html

```diff
@@ -9,11 +9,19 @@
       <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
     {% endif %}
     {% if tailored_results %}
-      {{ govukBackLink({
-        'classes': "back_link_other_region",
-        'text': BACK_LABEL,
-        'href': "/check?categories="+category_code,
-      }) }}
+      {% if request.GET.getlist('categories')|length > 1 %}
+        {{ govukBackLink({
+          'classes': "back_link_other_region",
+          'text': BACK_LABEL,
+          'href': "/check?categories="+category_code+"&sub-categories="+sub_category_code,
```

Comment:
might feel like I am banging on about this if you decide to not change it, but if you do, here is another plural `sub-categories`